web-client/src/business/chambers/getJudgesChambers.ts

```diff
@@ -203,6 +205,12 @@ const JUDGES_CHAMBERS: Record<string, JudgeInfo> = omitBy(
       phoneNumber: '(202) 521-0778',
       section: 'vasquezsChambers',
     },
+    WAYS_CHAMBERS_SECTION: {
+      judgeFullName: 'Kashi Way',
+      label: 'Way’s Chambers',
+      phoneNumber: '', // TODO
```

Comment:
This is unknown for the time being and should be filled in once we have the information.